Create 2019-11-30-paper.GraphClust2.md (#396)
usegalaxy-eu/website: _posts/2019-11-30-paper.GraphClust2.md

---
site: freiburg
title: "GraphClust2: Annotation and discovery of structured RNAs with scalable and accessible integrative clustering"
tags: [paper]
doi: "10.1093/gigascience/giz150"
supporters:
 - denbi
 - elixir
---

> Going back to the roots of the [Prof. Backofen lab](http://www.bioinf.uni-freiburg.de/?en), our colleague Milad has published a paper about annotation and discovery of structured RNAs with scalable and accessible integrative clustering with GraphClust2. We are happy that we could support Milad to integrate GraphClust2 into Galaxy and make it available to our RNA community. Congrats to Milad and colleagues!

Reviewer: ```suggestion
Going back to the roots of the [Prof. Backofen lab](http://www.bioinf.uni-freiburg.de/?en), our colleague Milad has published a paper on annotation and discovery of structured RNAs with scalable and accessible integrative clustering with GraphClust2. We are happy that we were able to support Milad to integrate GraphClust2 into Galaxy and make it available to our RNA community. Congratulations to Milad and colleagues!
```
